### ISSUE TITLE
404 error for existing entries

### ISSUE THREAD

**Issue by samosafuz** [papyri/idp.data#258]
Over the weekend I edited [ddbdp/p.lond;2;425](https://papyri.info/ddbdp/p.lond;2;425), following a tip from a Twitter user. It had not been aggregating properly with [ddbdp/sb;22;15624](https://papyri.info/ddbdp/sb;22;15624): its idnos were incomplete and neither file's DDB dummy header pointed to the other.

I thought I had updated everything correctly. After finalization, however, navigating to [p.lond;2;425](https://papyri.info/ddbdp/p.lond;2;425) produces a 404 error, as capped below.

<img width="734" alt="Screen Shot 2021-08-17 at 9 01 30 AM" src="https://user-images.githubusercontent.com/61889129/129730279-f1255d4e-fd69-45ec-88af-0ad4a209f399.png">

This is not the first time I've encountered 404 errors for existing entries: recently, a research assistant transcribed [dclp/131625](https://papyri.info/dclp/131625). Although the file [exists in GitHub](https://github.com/papyri/idp.data/blob/master/DCLP/132/131625.xml), it cannot be found in PN at the moment, as capped below.

<img width="730" alt="Screen Shot 2021-08-17 at 9 04 35 AM" src="https://user-images.githubusercontent.com/61889129/129730830-a24b69b2-d772-4655-8e76-67524a373fce.png">

Obviously, one can't 'open in editor' a file that can't be found. I had assumed after what happened with the London text that I'd simply have to wait for a re-index to clear the issue up; the case of 131625 makes me think that something else is to blame.

**Comment by samosafuz** [comment]
This issue is now affecting `tm;;60687` as well; I finalized that text a month ago, and can confirm that it displayed correctly after finalization. But now navigating to it in [DCLP](http://papyri.info/dclp/60687) produces the same 404 error described above.

**Comment by jcowey** [comment]
https://papyri.info/dclp/60946
is also not working after
https://github.com/papyri/idp.data/commit/50a22d30c897fc7bbbcaa145f2c0aabf34aa27d7
possibly because
https://github.com/papyri/idp.data/commit/36359a9da55a0d5498420239eda6f86b1186f1bd
had not gone through previously.

**Comment by samosafuz** [comment]
I'm noticing the same error now with [https://papyri.info/ddbdp/chr.mitt;;191](https://papyri.info/ddbdp/chr.mitt;;191). The file exists in GitHub and can be located by browsing DDB, but produces a 404 error when clicked on or navigated to.

<img width="951" alt="Screen Shot 2021-09-20 at 8 27 04 AM" src="https://user-images.githubusercontent.com/61889129/134002095-efe76aed-1ee1-4e47-bf3c-3f9392027519.png">


<img width="1128" alt="Screen Shot 2021-09-20 at 8 27 42 AM" src="https://user-images.githubusercontent.com/61889129/134002211-4e7bdad9-93c4-45be-b0bb-9c5e31109f9f.png">

**Comment by samosafuz** [comment]
This is perhaps unrelated to this ticket, but I'm including it here anyway. Yesterday my pull request #259 was merged, but none of the changes or new files (which appear in GitHub) are currently displaying in PN. 

I noticed after the merge yesterday that sync was taking a very long time (i.e., as of 9PM last night, the [status indicator](https://papyri.info/sync/?action=status) was still registering 'false', with a 10:13AM start time for publishing new files). The pull only modified 32 files -- that shouldn't take 11+ hours! 

Perhaps this entire issue has identified something malfunctioning in sync.
